projects/nae-example-app/src/app/doc/public-single-task-view/public-single-task-view.component.ts

```diff
@@ -51,27 +56,27 @@ const localAllowedNetsServiceFactory = (factory: AllowedNetsServiceFactory, rout
 
 const processServiceFactory = (userService: UserService, sessionService: SessionService, authService: AuthenticationService,
```

Comment:
i am once again asking about extracting these public service factories into library methods.

Has a task been created for this?
Why was it not done as a part of this task?

Reply:
resolved in a personal conversation - this will be done in a separate time once time allows for it